src/build.common.props

```diff
@@ -45,11 +45,4 @@ whether they use the VS 2017 project system or the old project system.
   <ItemGroup>
     <Compile Include="$(MsBuildThisFileDirectory)CommonAssemblyInfo.cs" />
   </ItemGroup>
-
-  <Target BeforeTargets="PrepareForBuild" Name="CreateVersionConstantsFile">
-    <Exec Command="powershell -ExecutionPolicy RemoteSigned $(MsBuildThisFileDirectory)..\scripts\New-VersionConstantsFile.ps1 $(IntermediateOutputPath) $(AssemblyName)" />
-    <ItemGroup>
-      <Compile Include="$(IntermediateOutputPath)VersionConstants.cs" />
-    </ItemGroup>
-  </Target>
```

Comment:
We do this only in Sarif.csproj now. And, since we generate the file in the source directory, you'll note that in Sarif.csproj, we don't need to mention the `<Compile>` item for VersionConstants.cs. (In fact if you do mention it, the project system complains that it's a duplicate item, because it already knows about it.)